javascript/javascript3/week3/app.js

```diff
@@ -15,3 +15,53 @@ context.strokeStyle = "#003300";
 context.stroke();
 
 //class
+
+class Circle {
+  constructor(x, y, r, startAngle, endAngle, fillColor) {
+    this.x = x;
+    this.y = y;
+    this.r = r;
+    this.startAngle = startAngle;
+    this.endAngle = endAngle;
+    this.fillColor = fillColor;
+  }
+
+  draw() {
+    context.beginPath();
+    context.arc(this.x, this.y, this.r, this.startAngle, 2 * Math.PI);
+    context.fillStyle = this.fillColor;
+    context.fill();
+    context.lineWidth = 5;
+    context.strokeStyle = "#003300";
+    context.stroke();
+  }
+}
+
+const c1 = new Circle(50, 50, 20, 0, 2 * Math.PI, "#000000");
+c1.draw();
+
+//running every 100ms
+function generateRandom(maxLimit = 100) {
+  let rand = Math.random() * maxLimit;
+  console.log(rand); // say 99.81321410836433
+
+  rand = Math.floor(rand); // 99
+
+  return rand;
+}
```

Comment:
good idea to create function for random numbers and call it for height, width and radius!